src/components/Button.js

```diff
@@ -17,16 +17,17 @@ class Button extends React.Component {
   }
 
   render() {
-    const { cssClass, buttonName } = this.props;
+    const { color, size, buttonName } = this.props;
     return (
-      <button className={`button ${cssClass}`} onClick={this.handleClick} type="button">{buttonName}</button>
+      <button className={`btn ${size} bg-${color || 'orange'}`} onClick={this.handleClick} type="button">{buttonName}</button>
     );
   }
 }
 
 Button.propTypes = {
   buttonName: PropTypes.string,
-  cssClass: PropTypes.string,
+  size: PropTypes.string,
+  color: PropTypes.string,
   clickHandler: PropTypes.func,
 };
```

Comment:
`size` (i suppose is what the [requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/WideProps.png) calls `wide`), it should be Boolean

Reply:
I will update that